includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-students.php

```diff
@@ -75,9 +75,21 @@ public function get_items( array $filters ): array {
 		}
 
 		$wp_user_search = new WP_User_Query( $query_args );
+		remove_action( 'pre_user_query', [ $this, 'add_orderby_custom_field_to_user_query' ] );
+
+		$wp_all_users       = new WP_User_Query(
+			array_merge(
+				$query_args,
+				[
+					'number' => -1,
+					'fields' => 'ids',
+				]
+			)
+		);
+		$this->all_item_ids = $wp_all_users->get_results();
+
```

Comment:
I think this could be reverted as well.

Reply:
Hey I reverted it but as a change, I didn't use git revert since this change is upon the other change and it would be too big of a mess reverting multiple things a top of one another
[7b0a333ac8bdfe493fa3152cc7ce04f3ee7210d1](https://github.com/Automattic/sensei/pull/5091/commits/7b0a333ac8bdfe493fa3152cc7ce04f3ee7210d1)